src/generic-bioengine-app.imjoy.html

```diff
@@ -260,10 +271,7 @@
         
 
         info = f"""
-        <div>
-            <div class='media-left'>
-                <figure class='image is-64x64'><img src='https://raw.githubusercontent.com/imjoy-team/bioimage-io-resources/master/src/bioengine-icon.svg' alt='Image'></figure>
-            </div>
+        <div class='box'>
```

Comment:
I am not sure how these changes happened (I just edited on github...). If this is important you probably shouldn't merge the whole PR, just copy the suggestions for the better shape messages above.

Reply:
Don't worry, I fixed it.